ipatests/test_integration/test_passkey.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2020  FreeIPA Contributors see COPYING for license
+#
+
+"""
+Tests to verify Passkey Feature.
+"""
+
+from __future__ import absolute_import
+
+import os
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+
+def setup_umockdev(host):
+    tasks.install_packages(host,
```

Comment:
This method could be a fixture called by the test test_passkey_register_ipa_user, and should cleanup the .bashrc file at the end of the test. You can use bashrc = tasks.FileBackup / yield / bashrc.restore:
```
bashrc = tasks.FileBackup(host, BASHRC_CFG)
contents = host.get_file_contents(BASHRC_CFG, encoding="utf-8")
add_vars = textwrap.dedent("""
    export HIDRAW=/dev/hidraw1
    export LD_PRELOAD=/opt/random.so
""")
new_content = contents + add_vars
host.put_file_contents(BASHRC_FILE, new_content)
yield
bashrc.restore()